kadi/commands/observations.py

```diff
@@ -417,10 +435,15 @@ def get_observations(start=None, stop=None, *, obsid=None, scenario=None, cmds=N
         Scenario
     :param cmds: CommandTable, None
         Use this command table instead of querying the archive.
+    :param starcat_date: CxoTime-like, None
+        Date of the observation's star catalog
     :returns: list of dict
         Observation parameters for matching observations.
     """
     from kadi.commands.commands_v2 import get_cmds
+    if starcat_date is not None:
+        start = starcat_date if start is None else start
+        stop = CxoTime(starcat_date) + 7 * u.day if stop is None else stop
```

Comment:
Why 7 days? I think this could be 1 sec (technically anything over 1 msec).

Reply:
That is a good question. I don't know why I used 7 days. Couldn't it be equal to start? (because the start/stop interval is closed on both ends, isn't it?)

Reply:
Ok. Fyi get_cmds is like slice and it is open for stop.